src/openMVG/matching_image_collection/E_ACRobust.hpp

```diff
@@ -102,11 +102,7 @@ struct GeometricFilter_EMatrix_AC
     const std::pair<double,double> ACRansacOut =
       ACRANSAC(kernel, vec_inliers, m_stIteration, &m_E, upper_bound_precision);
 
-#ifdef HAVE_CCTAG
-    if (vec_inliers.size() > KernelType::MINIMUM_SAMPLES)  {
-#else
-    if (vec_inliers.size() > KernelType::MINIMUM_SAMPLES *2.5)  {  
-#endif
+    if (vec_inliers.size() > KernelType::MINIMUM_SAMPLES * OPENMVG_MINIMUM_SAMPLES_COEF)  {  
```

Comment:
@pmoulon i was thinking of something about that even inside the ACRansac in order to validate the first model found by normal ransac (if threshold != inf), as it could be the case (very unlucky, though) that we got stuck in a bad model: imagine the case on lines, it could be that the outliers fit a second line with enough support, we assume that model and we keep drawing models from those outliers. Unless a couple of those points also belong to the real line, we get stuck on a bad model. Maybe the NFA will prevent to take/validate that model but still we will exit without the good model.